pages/about/index.vue

```diff
@@ -14,15 +14,15 @@
       </div>
       <div class="text-block">
         <h3>{{$t('about_page.description.heading3')}}</h3>
-        <p>{{$t('about_page.description.para3')}}</p>
+        <p><NuxtLink to="/companies">{{ $t('about_page.description.para3') }}</NuxtLink></p>
```

Comment:
I think it looks better if only the word catalog is the link instead of the entire line.

Reply:
I didn't have a good solution for that way, but now I thought of one os can fix it!